src/Elasticsearch.Net/Transport/Pipeline/RequestData.cs

```diff
@@ -113,23 +113,13 @@ private string CreatePathWithQueryStrings(string path, IConnectionConfigurationV
 			return path;
 		}
 
-
 		protected bool Equals(RequestData other) =>

```

Comment:
I was thinking exactly the same thing 😄 The Equality and HashCode are only used for this. 

I'll update now